app/Http/Controllers/Admin/PetShop/OwnerCrudController.php

```diff
@@ -99,4 +100,50 @@ protected function setupUpdateOperation()
     {
         $this->setupCreateOperation();
     }
+
+    protected function setupShowOperation()
+    {
+        $this->setupListOperation();
+
+        Widget::add([
+            'type'       => 'datatable',
+            'controller' => 'App\Http\Controllers\Admin\PetShop\PetCrudController',
+            'name'       => 'pets_crud',
+            'section'    => 'after_content',
+            'wrapper'    => ['class' => 'mt-3'],
+            'content'    => [
+                'header' => 'Pets for this owner',
+                // COULD-DO: maybe add support for a subheader?
+                // 'subheader' => 'This is a list of all pets owned by this owner.',
+            ],
+            // MUST-DO: How the fuck do I make this only show related pets?!?!
+            // 'configure' => function ($crud, $parent) {
+            //     // only show the pets of this owner (owner is an n-n relationship)
+            //     if ($parent) {
+            //         $crud->addClause('whereHas', 'owners', function ($query) use ($parent) {
+            //             $query->where('id', $parent->id);
+            //         });
+            //     }
+            // },
+            // SHOULD-DO: how do I make a new entry automatically related to the owner?
```

Comment:
How does someone make the "add" button automatically select a value for an input? We might not need this to become A FEATURE, but we need to understand how someone can do that NOW. So let's:
- [ ] add that to the documention;
- [ ] regroup to decide if it's intuitive/easy enough;